ibc_data/conditions.csv

```diff
@@ -0,0 +1,377 @@
+PreferenceHouses,house_constant,classify the level of pleasantness of a house displayed on the screen in terms of willingness to live in that house. visual evaluation of an item vs. fixation
+PreferenceHouses,house_linear,classify the level of pleasantness of a house displayed on the screen in terms of willingness to live in that house. visual preference vs. no preference
+PreferenceHouses,house_quadratic,classify the level of pleasantness of a house displayed on the screen in terms of willingness to live in that house. confidence in preference vs. no confidence
+TheoryOfMind,belief,read a false-belief story
+TheoryOfMind,photo,read a false-photograph story
+EmotionalPain,physical_pain,read story about fictional characters suffering from physical pain
+EmotionalPain,emotional_pain,read story about fictional characters suffering from emotional pain
+PainMovie,movie_pain,watch movie-scene wherein characters experience physical pain
+PainMovie,movie_mental,"watch movie-scene wherein characters experience changes in beliefs, desires, and/or emotions"
+VSTM,vstm_linear,"judge whether any bar changed orientation within two consecutive displays of bar sets on the screen, response to constant numerosity vs. fixation"
+VSTM,vstm_constant,"judge whether any bar changed orientation within two consecutive displays of bar sets on the screen, esponse to numerosity vs. fixation"
+VSTM,vstm_quadratic,"judge whether any bar changed orientation within two consecutive displays of bar sets on the screen, response to numerosity interaction vs. fixation"
+Enumeration,enumeration_linear,enumeration response to constant numerosity vs. fixation
+Enumeration,enumeration_constant,enumeration response to numerosity vs. fixation
+Enumeration,enumeration_quadratic,enumeration response to numerosity interaction vs. fixation
+Self,instructions,presentation of a question related to the succeeding block
+Self,encode_self,"judge with overt response whether or not a certain adjective, displayed on the screen, qualifies oneself. self-reference encoding vs. other-reference encoding"
+Self,encode_other,"judging with overt response whether a certain adjective, displayed on the screen, qualifies someone else, other-reference encoding vs. fixation"
+Self,recognition_self_hit,"successful recognition with an overt response of an adjective, displayed on the screen, as having been already presented during one “self-reference encoding” trial of the preceding encoding phase, self-reference recognition vs. other-reference recognition"
+Self,recognition_other_hit,"successful recognition with an overt response of an adjective, displayed on the screen, as having been already presented during one “other-reference encoding” trial of the preceding encoding phase, other-reference recognition vs. fixation"
+Self,correct_rejection,

```

Comment:
missing description

Reply:
Yes, I did not add the description to correct_rejection because that is not a condition but it's specified as a contrast.

Reply:
I left only conditions' specifications, well described, removed contrasts.